features/admin.tenants.v1/components/forms/add-tenant-wizard-form.tsx

```diff
@@ -349,6 +382,21 @@ export const AddTenantWizardForm: FunctionComponent<AddTenantWizardFormPropsInte
                     width={ 16 }
                     data-testid={ `${ testId }-type-input` }
                 />
+                { isCentralDeploymentEnabled  ? (
+                    <Field.Dropdown
+                        ariaLabel="Region"
+                        name="deploymentUnit"
+                        label={ t("tenants:deploymentUtits.label") }
+                        placeholder = { t("tenants:deploymentUtits.placeholder") }
+                        required={ true }
+                        options={ deploymentUnitOptions }
+                        readOnly={ true }
```

Comment:
if this is readonly, end users may not be able to select the region, might they?

Reply:
It worked locally with above configurations

Reply:
Changed it